corehq/apps/linked_domain/management/commands/link_app_to_remote.py

```diff
@@ -34,3 +35,4 @@ def handle(self, master_id, linked_id, url_base, domain, username, api_key, **op
 
         linked_app = LinkedApplication.get(linked_id)
         link_app(linked_app, domain, master_id, remote_details)
+        update_linked_app(linked_app, master_id, 'system')
```

Comment:
I think it's a bit confusing if the command that is suppose to "link app to remote" also secretly updates it in the end. I think generating a prompt in the end to user asking if they want to update the app as well or rather do it manually, in which case we just tell them where to do it would be better?

Reply:
The user can't do it manually later. Without doing an initial pull, after this command finishes, the linked app doesn't know what its master app is.

Reply:
So this HAS to be done here, without it the process isn't complete.

Cool. Thanks for clarifying.